Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2005,15 +2059,22 @@ def slack_send_file(_channel: str | None = None, _channel_id: str = '', _entry_i
         'name': file_path['name'],
         'comment': comment
     }
+
+    error_message = 'Could not send the file to Slack.'
     try:
         response = slack_send_request(to, channel, group, thread_id=thread_id, file_dict=file_dict, channel_id=channel_id)
         if response:
-            demisto.results('File sent to Slack successfully.')
+            return_results(CommandResults(readable_output='File sent to Slack successfully.'))
         else:
-            demisto.results('Could not send the file to Slack.')
-    except SlackApiError as e:
-        if 'method_deprecated' in str(e):
-            raise DemistoException('Command slack-send-file isn\'t available for newly created apps (from May 8, 2024).')
+            raise DemistoException(message=error_message)
+    except SlackApiError as api_error:
```

Comment:
```suggestion
    except SlackApiError as e:
```

Reply:
Fixed in commit 7c5b1ee65136fc655e92a3456301506de163b9e6